### ISSUE TITLE
OpenSSH: SFTP 'permission denied' when moving a file. 

### ISSUE THREAD

**Issue by Neo-Oli** [termux/termux-packages#293]
Hi

I am having an issue with OpenSSH. When I connect to Termux via sftp (with FileZilla) I can download, edit, upload and delete files just fine, but I cannot move or rename them:
```
Command:	mv "color.css" "color.all-white.css"
Error:	mv /storage/emulated/0/drive/internal/oliverse/css/color.css /storage/emulated/0/drive/internal/oliverse/css/color.all-white.css: permission denied
```

**Comment by fornwall** [comment]
Is that also in `$HOME`, or only under shared storage?

**Comment by Neo-Oli** [comment]
It is alsp happening in $HOME as well

**Comment by fornwall** [comment]
Nice find - this is now fixed in version `7.2p2-3` of the `openssh` package!
